app/src/main/java/com/kanedias/vanilla/coverfetch/CoverShowActivity.java

```diff
@@ -242,10 +242,10 @@ public boolean onOptionsItemSelected(MenuItem item) {
      *     extras from artwork response (if found) or with original intent
      * </pre>
      */
-    private static boolean pluginInstalled(Context ctx, String pkgName) {
+    private static boolean pluginInstalled(Context ctx) {
```

Comment:
Nope, this will add confusion. This function can possibly be reused for other plugins, so please leave it as it was.